spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,8 @@
+if $facts['os']['name'] == 'Ubuntu' {
+  # Facter < 4 needs lsb-release for os.distro.codename
+  if versioncmp($facts['facterversion'], '4.0.0') <= 0 {
+    package { 'lsb-release':
+      ensure => installed,
+    }
+  }
+}
```

Comment:
I'm not a fan of adding this to every single module that needs it.
At least if we do we should probably add it to a file that can be easily removed with modulesync?

Reply:
If it's common, perhaps it's something that should live in voxpupuli-acceptance? Right now I don't have time to work on that, but can review things if needed.